source-academy/frontend
# Game: Remove Object Placement feature from Game Simulator

### Description



Partially addresses #2806. This PR will be in draft for now until we have finalized the decision to remove the Object Placement feature.

This PR removes the Object Placement feature completely from the Game Simulator. As discussed in the linked issue, the Object Placement feature is currently not in use by the story-writers (myself), and is inaccessible to the students. Hence, there are no active users, and may need to be retired.

While there certainly is potential for the Object Placement feature to play a role in the future (e.g. being able to generate completely new asset combinations for the game and the CMR feature), it can be retired for now, and we can re-implement it with only the necessary features when needed. 

### Type of change


*This would be considered breaking change by definition; however, the rest of the Game Simulator is quite independent from the Object Placement feature, and the removal of this feature should not cause any major changes to the other functionalities.*

### How to test



Build and deploy the frontend, and check that all other functionalities continue to work as expected.

### Checklist
